app/hellodjango/locations/helpers/django_admin.py

```
@@ -0,0 +1,94 @@
+from django.db.models import Max
+
+from locations.models import United_States_Address
+
+logger = logging.getLogger("django")
+
+
+def get_random_object(django_model):
+    """Pick a random object from a Django model's table by random PK lookup."""
+    max_id = django_model.objects.all().aggregate(max_id=Max("id"))["max_id"]
+    while True:
+        pk = random.randint(1, max_id)
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)